mmpose/core/evaluation/top_down_eval.py

```diff
@@ -250,6 +250,69 @@ def _taylor(heatmap, coord):
     return coord
 
 
+def post_dark(coords, batch_heatmaps, kernel=3):
+    """DARK post-pocessing.
+
+    Note:
+        batch_size: B
```

Comment:
Since this is note, `batch_size` -> `batch size`

Reply:
anyway, this is not important

Reply:
ok， i will update this in the next commit